staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics.go

```diff
@@ -39,6 +39,15 @@ var (
 		[]string{"resource"},
 	)
 
+	eventsCounter = metrics.NewCounterVec(
```

Comment:
@dgrisonnet @logicalhan - can you please take a look at proposed new metrics and review it?

Reply:
As a general comment for this file, I think we should set `Namespace: "apiserver"` and `Subsystem: "watch_cache"` in all the metric options. That would result in having all the metrics prefixed by `apiserver_watch_cache` which would make the naming of the metrics related to the watch cache more consistent and thus easier to use.

Reply:
I updated the newly added metrics.

What are our rules for touching existing metrics? Given they are all alpha - can we just rename them as you suggest?

Reply:
[If we can change that - I think I would do that in a separate PR anyway with a clear release note - just please confirm we want this.]

Reply:
Considering that these metrics are ALPHA, we could theoretically rename them right away, but since some of them have been around for a while, it might be safer to have them go through a 2 releases deprecation cycle and introduce new metrics as ALPHA. cc @logicalhan for confirmation

Yeah that would make complete sense to do that in a separate PR :+1:

Reply:
OK - thanks. I will open a separate PR for that in the next days.